plugins/woocommerce/tests/e2e-pw/tests/api-tests/customers/customers-crud.test.js

```diff
@@ -259,9 +289,10 @@ test.describe(
 				// update customer names (billing and shipping) to Jane
 				// (these were initialised blank, only regular first_name was populated)
 				const response = await request.put(
-					`/wp-json/wc/v3/customers/${ subscriberUserId }`,
+					`./wp-json/wc/v3/customers/${ subscriberUserId }`,
 					{
 						data: {
+							first_name: 'Jane',
```

Comment:
We are not sure the subscriber created in the beforeAll is used, and this means the first_name can be different when we're checking it down the line.